app/services/bill-runs/two-part-tariff/review-bill-run.service.js

```diff
@@ -12,17 +12,17 @@ const ReviewBillRunPresenter = require('../../../presenters/bill-runs/two-part-t
  * Orchestrates fetching and presenting the data needed for the review bill run page
  *
  * @param {String} id The UUID for the bill run to review
- * @param {Object} payload The `request.payload` containing the filter data. This is only passed to the service when
- * there is a POST request, which only occurs when a filter is applied to the results.
  * @param {Object} yar - The Hapi `request.yar` session manager passed on by the controller
  *
  * @returns {Promise<Object>} An object representing the `pageData` needed by the review bill run template. It contains
  * details of the bill run and the licences linked to it as well as any data that has been used to filter the results.
  */
-async function go (id, payload, yar) {
-  const filterIssues = payload?.filterIssues
-  const filterLicenceHolder = payload?.filterLicenceHolder
-  const filterLicenceStatus = payload?.filterLicenceStatus
+async function go (id, yar) {
+  const filters = yar.get('filters')
```

Comment:
Just to avoid any collisions in the future I think we should give this a less generic name.

```suggestion
  const filters = yar.get('reviewFilters')
```